Source/Relativity.Test.Helpers/Relativity.Test.Helpers/ArtifactHelpers/DocumentHelper.cs

```diff
@@ -0,0 +1,73 @@
+﻿using Relativity.API;
+using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Data.SqlClient;
+using Newtonsoft.Json;
+using Relativity.Test.Helpers.ArtifactHelpers.Interfaces;
+using Relativity.Test.Helpers.Exceptions;
+using TestHelpersKepler.Interfaces.TestHelpersModule.v1.Models;
+
+namespace Relativity.Test.Helpers.ArtifactHelpers
+{
+	/// <summary>
+	/// 
+	/// Helpers to interact with Documents in Relativity
+	/// 
+	/// </summary>
+
+	public class DocumentHelper : IDocumentHelper
+	{
+		private IHttpRequestHelper _httpRequestHelper;
+		public DocumentHelper(IHttpRequestHelper httpRequestHelper)
+		{
+			_httpRequestHelper = httpRequestHelper;
+		}
+
+		public string GetDocumentIdentifierFieldColumnName(int fieldArtifactTypeID, int workspaceID)
+		{
+			try
+			{
+				const string routeName = "GetDocumentIdentifierFieldColumnNameAsync";
```

Comment:
you should probably move all the endpoint names to a TestConstants class so they are all together in one place. In case we need to update them in future.

Reply:
This should be a separate task

Reply:
This should be a separate task